R/readQFeaturesFromDIANN.R

```diff
@@ -61,6 +61,10 @@
 ##' ## using default arguments
 ##' readQFeaturesFromDIANN(x)
 ##'
+##' ## use the precursor identifier as assay rownames
+##' readQFeaturesFromDIANN(x) |>
+##'     rownames()
```

Comment:
I think the example is incomplete, I would expect `fnames` to be used here.

Reply:
Indeed. Fixed.